uber/templates/preregistration/badge_choice.html

```diff
@@ -106,6 +100,12 @@ <h2 style="margin-top:10px"> Choose a Badge </h2>
     {% endif %}
     </tr>
 {% endif %}
```

Comment:
A slightly cleaner implementation would be to make ``ONE_DAYS_ENABLED`` part of the logic in ``DISPLAY_ONSITE_BADGES``.  It's definitely okay as-is though.